release-notes/v1.1.0-rc1.yaml

```diff
@@ -0,0 +1,229 @@
+      Added Support for using the Listener port as a the Container port in EnvoyProxy CRD
+      Added Support for OpenTelemtry Sink Export Settings in EnvoyProxy CRD
+      Added Support for Backend Client Certificate Authentication in EnvoyProxy CRD
+      Added Support for Backend TLS Settings in EnvoyProxy CRD
+      Added Support for HTTP Filter Ordering in EnvoyProxy CRD
+      Added Support for gRPC Access Log Service (ALS) Sink in EnvoyProxy CRD
+      Added Support for OpenTelelemetry Sinks as a BackendRef in EnvoyProxy CRD
+      Added Support for User-Provided name for generate Kubernetes resources in EnvoyProxy CRD
+      Added Support for Per-Endpoint stats in EnvoyProxy CRD
+      Added Support for Targeting SectionNames in ClientTrafficPolicy CRD
+      Added Support for Preserving X-Request-ID header in ClientTrafficPolicy CRD
+      Added Support for Using Downstream Protocol in Upstream connections in ClientTrafficPolicy CRD
+      Added Support for HTTP/2 settings in ClientTrafficPolicy CRD
+      Added Support for Connection Buffer Size Limit in ClientTrafficPolicy CRD
+      Added Support for HTTP Health Check in ClientTrafficPolicy CRD
+      Added Support for Optionally requiring a Client Certificate in ClientTrafficPolicy CRD
+      Added Support for Headers with Underscores CRD in ClientTrafficPolicy CRD
+      Added Support for XFCC header processing in ClientTrafficPolicy CRD
+      Added Support for TCP Listener Idle Timeout in ClientTrafficPolicy CRD
+      Added Support for IdleTimeout in ClientTrafficPolicy CRD
+      Added Support for Connection Limits in ClientTrafficPolicy CRD
+      Added Support for additional OIDC settings related to Resource, Token and Cookie in SecurityPolicy CRD
+      Added Support for Optionally requiring a JWT in SecurityPolicy CRD
+      Added Support for BackendRefs for Ext-Auth in SecurityPolicy CRD
+      Added Support for Authorization in SecurityPolicy CRD
+      Added Support for Ext-Auth failOpen in SecurityPolicy CRD
+      Added Support for Loadbalancer Cookie Consistent Hashing in BackendTrafficPolicy CRD
+      Added Support for Disabling X-RateLimit headers in BackendTrafficPolicy CRD
+      Added Support for Connection Buffer Size Limit in BackendTrafficPolicy CRD
+      Added Support for Loadbalancing Consistent Hash Table Size in BackendTrafficPolicy CRD
+      Added Support for Loadbalancing Header Hash Policy in BackendTrafficPolicy CRD
+      Added Support for Cluster Connection Buffer Size Limit in BackendTrafficPolicy
+      Added Support for more Rate Limit Rules in BackendTrafficPolicy CRD
+      Added Support for WASM extension in EnvoyExtensionPolicy CRD
+      Added Support for External Processing extension in EnvoyExtensionPolicy CRD
+      Removed Status Print Column from xPolicy CRDs
+
+
+    breaking-change: |
+      Gateway-API BackendTLSPolicy v1alpha3 is incompatible with previous versions of the CRD
```

Comment:
we need to add some Upgrade notes in here similar to https://gateway-api.sigs.k8s.io/guides/?h=release+not#backendtlspolicy

Reply:
Any thoughts on how to include this? Just add a reference to the GW-API docs in this comment? Another "Upgrade Guide" section in the release notes? should we create a proper Upgrade Guide doc for each release?

Reply:
thanks for adding it to the Quickstart, can we also add the  link in here ?